### DISCUSSION TITLE
Oddly low channel CPA within the new Budget Allocator (Robyn V3.10)

### DISCUSSION THREAD

**Discussion by mayank-fh** [facebookexperimental/Robyn#661] [Q&A] [answered]
In the channel breakup of the budget allocator, I notice the initial/bounded CPA turned out be oddly low compared it with the overall CPA. Can someone help me know the logic behind this. Thanks !
![image](https://user-images.githubusercontent.com/119480461/227453489-53dafcb0-33bb-4239-86ce-3cdd1edbcc91.png)

**Comment by gufengzhou** [comment, marked as answer]
This should be solved with the latest version. Can you please update and retry

**Reply by mayank-fh** [reply]
Worked. Thanks !

**Reply by creatorvision** [reply]
Working fine for our code  and for the use case of @mayank-fh
